src/Config.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\HttpExtractor;
+
+use Keboola\HttpExtractor\Config\ConfigDefinition;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+
+class Config
+{
+    /** @var array */
+    private $config;
+
+    private function __construct(
+        array $config
+    ) {
+        $this->config = $config;
+    }
+
+    public static function fromFile(string $configPath): self
+    {
+        $contents = file_get_contents($configPath);
+        $decoder = new JsonDecode(true);
+        $config = $decoder->decode($contents, JsonEncoder::FORMAT);
+
+        return self::fromArray($config['parameters']);
+    }
+
+    public static function fromArray(array $config): self
```

Comment:
nemusi byt public

Reply:
Když na to koukám dneska s odstupem, tak vidím, že to vlastně může být přímo v ctoru tohle. Už jen proto, že ta metoda má stejné parametry a dělá to ze stejných dat.